<launch>
    <!-- period in seconds -->
    <arg name="period" default="10"/>
    <arg name="dt" default="0.001"/>
    <!-- deviation amplitude in m -->
    <arg name="amplitude_x" default="0.01"/>
    <arg name="amplitude_y" default="0.01"/>

    <node pkg="rcm_eval" ns="lbr" name="rcm_deviation_node" type="rcm_deviation_node.py" output="screen">
        <param name="period" value="$(arg period)"/>
        <param name="dt" value="$(arg dt)"/>
        <param name="amplitude_x" value="$(arg amplitude_x)"/>
        <param name="amplitude_y" value="$(arg amplitude_y)"/>
    </node>
</launch>
